packages/protocol/test/common/multisig.ts

```diff
@@ -133,18 +203,32 @@ contract('MultiSig', (accounts: any) => {
     it('should not allow an external account to add an owner', async () => {
       await assertRevert(multiSig.addOwner(accounts[3]))
     })
+
+    it('should not allow adding the null address', async () => {
```

Comment:
I don't think this is testing what we'd like it to. It's failing because it's not being called by the `multisig` address, and not because it's adding the `NULL_ADDRESS`.

Reply:
true, neither was this well thought through in the old tests, so I changed it